frontend/src/metabase/nav/containers/MainNavbar/types.ts

```diff
@@ -4,7 +4,7 @@ export type SelectedEntityItem = {
 };
 
 export type SelectedNonEntityItem = {
-  type: "unknown";
+  type: "homepage" | "unknown";
```

Comment:
I think we should just use `"unknown"` for the homepage [the same way we do for the "browse link":](https://github.com/metabase/metabase/blob/master/frontend/src/metabase/nav/containers/MainNavbar/MainNavbarView.tsx#L120)

```jsx
<HomepageLink isSelected={isMiscLinkSelected && selectedItem.url === "/"} />`
```

Guess `"unknown"` is just a bad name here and something like `"misc"` / `"non-entity"` would work better